src/config.rs

```diff
@@ -27,7 +27,7 @@ impl Config {
             .set_rest_api_endpoint("https://testnet.binance.vision")
             .set_ws_endpoint("wss://testnet.binance.vision")
             .set_futures_rest_api_endpoint("https://testnet.binancefuture.com")
-            .set_futures_ws_endpoint("wss://testnet.binancefuture.com")
+            .set_futures_ws_endpoint("wss://fstream.binancefuture.com")
```

Comment:
Apparently, fstream is for prod, stream is for testnet, can you confirm, and if so change this to stream.binancefuture.com for testing ?

Reply:
![Screenshot_2024-02-05_23-52-12](https://github.com/Igosuki/binance-rs-async/assets/6938607/0a3bddab-d9c1-4f4e-bb0b-1bf5db091101)